index.html

```diff
@@ -6,9 +6,224 @@
+            alt="Group of people around a table looking at different computer monitors">
+        </a>
+      </div>
+      <div class="col-xs-12 col-sm-9">
+        <h2 class="media-heading">Argentina 2015</h4>
+        <p>
+          Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque
+          laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi
+          architecto beatae vitae dicta sunt explicabo.
+        </p>
+      </div>
+    </div>
+
+    <div class="media row">
+      <div class="col-xs-offset-2 col-sm-offset-0 col-xs-8 col-sm-3">
+        <a href="img/deployments/ghana-2012.png" data-toggle="lightbox">
+          <img class="media-object" src="img/deployments/ghana-2012.png"
+            alt='Banner on blue building reads "Ghana Election 2012, Social Media Tracking Centre"'>
+        </a>
+      </div>
+      <div class="col-xs-12 col-sm-9">
+        <h2 class="media-heading">Ghana 2012</h4>
+        <p>
+          Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque
+          laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi
+          architecto beatae vitae dicta sunt explicabo.
+        </p>
+      </div>
+    </div>
+
+    <div class="media row">
+      <div class="col-xs-offset-2 col-sm-offset-0 col-xs-8 col-sm-3">
+        <a href="img/deployments/nigeria-2011.png" data-toggle="lightbox">
+          <img class="media-object" src="img/deployments/nigeria-2011.png"
+            alt="Two brown women point at computer screen showing red graphs">
+        </a>
+      </div>
+      <div class="col-xs-12 col-sm-9">
+        <h2 class="media-heading">Nigeria 2011</h4>
+        <p>
+          Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque
+          laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi
+          architecto beatae vitae dicta sunt explicabo.
+        </p>
+      </div>
+    </div>
+  </section>
+
+  <section class="design-principles color-block">
+    <h1>About Contact</h1>
+
+    <p>Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo.</p>
+
+    <p>Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo.</p>
+
+    <p>Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo.</p>
+  </section>
+
+  <footer>
+    <button class="btn">Request a demo</button>
+
+    <div class="logos">
+      <img src="img/logo.png" alt="Aggie Logo" />
```

Comment:
The logos at the bottom need to be replaced with actual logos for the 3 organizations. I will make that change.